FBSnapshotTestCase/FBSnapshotTestCase.m

```diff
@@ -7,8 +7,8 @@
  *
  */
 
-#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
-#import <FBSnapshotTestCase/FBSnapshotTestController.h>
+#import "Public/FBSnapshotTestCase.h"
```

Comment:
Curious about this change; can we not specify which files are Public and Internal via the `Package.swift`?

Reply:
I try specify, but in Package.swift the property `publicHeadersPath: ` have an unique path for file `.h` and conflict with file `.swift`